src/classes/CourseConSettingHealthCheckVMapper.cls

```diff
@@ -0,0 +1,352 @@
+        if (courseConSettingtHlthCheckGroupVModelsList[0].healthCheckItemList.size() > 0) {
+            return this.combineAccountHealthCheckGroupViewModels(courseConSettingtHlthCheckGroupVModelsList);
+        }
+        courseConSettingtHlthCheckGroupVModelsList.add(this.getStudentHealthCheckItems(courseConnSettingsModel, courseConnRTModelsById));
+        courseConSettingtHlthCheckGroupVModelsList.add(this.getFacultyHealthCheckItems(courseConnSettingsModel, courseConnRTModelsById));
+        courseConSettingtHlthCheckGroupVModelsList.add(this.getCourseConnectionRecordTypeUniquenessHealthCheckItems(courseConnSettingsModel));
+
+        return this.combineAccountHealthCheckGroupViewModels(courseConSettingtHlthCheckGroupVModelsList);
+
+    }
+
+    /**********************************************************************************
+    * @description Generates a HealthCheckGroupVModel comprised of the combined health 
+    * check results for Default Account, Administrative and Household accounts. 
+    * @param  healthCheckGroupVModelsList A list of HealthCheckGroupVModels
+    * @return A HealthCheckGroupVModel representing the health check results for 
+    * Default, Administrative and Household Accounts.
+    **********************************************************************************/ 
+    @TestVisible
+    private HealthCheckGroupVModel combineAccountHealthCheckGroupViewModels(List<HealthCheckGroupVModel> healthCheckGroupVModelsList) {
+
+        Integer hlthChkGroupPassedChecks = 0;
+        Integer hlthChkGroupTotalChecks = 0;
+
+        List<String> hlthChkGroupExpandedRowsList = new List<String>();
+        List<HealthCheckItemVModel> hlthChkGroupHlthChkItemsList = new List<HealthCheckItemVModel>();
+
+        for (HealthCheckGroupVModel hlthChkGroupVMod : healthCheckGroupVModelsList) {
+            hlthChkGroupPassedChecks += hlthChkGroupVMod.passedChecks;
+            hlthChkGroupTotalChecks += hlthChkGroupVMod.totalChecks;
+
+            hlthChkGroupExpandedRowsList.addAll(hlthChkGroupVMod.expandedRowsList);
+            hlthChkGroupHlthChkItemsList.addAll(hlthChkGroupVMod.healthCheckItemList);
+        }
+
+        HealthCheckGroupVModel hlthChkGroupVMod = new HealthCheckGroupVModel(Label.stgHCCourseConnectionsTitle,
+                                                                             Label.stgHealthCheckStatusPassed,
+                                                                             hlthChkGroupPassedChecks,
+                                                                             hlthChkGroupTotalChecks,
+                                                                             hlthChkGroupExpandedRowsList,
+                                                                             hlthChkGroupHlthChkItemsList);
+
+        if(hlthChkGroupVMod.passedChecks < hlthChkGroupVMod.totalChecks) {
```

Comment:
A quick note nothing big deal thou: I believe @m-kinkade is adding a updateHealthCheckStatus() method to HealthCheckGroupVModel